README.md

```diff
@@ -1,2 +1,97 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # TO-DO-LIST
-It simply lists the things that you need to do and allows you to mark them as complete, and it uses ES6 and Webpack.
+
+> Description the project.
+- It simply lists the things that you need to do and allows you to mark them as complete, and it uses ES6 and Webpack.
+- Single page website.
+- configure with linters.
+
+## Screenshot
+
+![readme](./assets/screenshoots/M1.png)
+![readme](./assets/screenshoots/M2.png)
+![readme](./assets/screenshoots/D1.png)
+![readme](./assets/screenshoots/D2.png)
```

Comment:
- [ ] Your screenshots links are broken , please fix it in order to make them appear in your readme and make it more professional :smile: